Makefile

```diff
@@ -1,20 +1,21 @@
-.PHONY: all tidy generate lint vet test coverage pushdoc
+.PHONY: all tidy generate lint test coverage pushdoc
 
 # Default "make" target to check locally that everything is ok, BEFORE pushing remotely
-all: lint vet test
+all: lint test
 	@echo "Done with the standard checks"
 
 tidy:
-	go mod tidy
+	@go mod tidy -go="1.23.8"
 
 generate: tidy
-	go get -u google.golang.org/protobuf/cmd/protoc-gen-go@v1.36.5
+	go get -u google.golang.org/protobuf/cmd/protoc-gen-go@v1.5.1
```

Comment:
I'm wondering if we really need to revert to such an old version?
protoc-gen-go and protoc-gen-go-grpc versions are not aligned, this seems likely unnecessary and potentially problematic 🤔
protoc-gen-go-grpc 1.5.1 was released >=2024.
protoc-gen-go 1.5.1 was released <= 2020